src/Aspire.Dashboard/Components/ResourcesGridColumns/StateColumnDisplay.razor

```diff
@@ -25,13 +25,5 @@
     }
 
     @Resource.State
-    <UnreadLogErrorsBadge UnviewedCount="@GetUnviewedErrorCount(Resource)" Resource="@Resource"/>
+    <UnreadLogErrorsBadge UnviewedErrorCounts="UnviewedErrorCounts" Resource="@Resource" />
```

Comment:
I made some changes to the unread errors component because I wanted to reuse it in the resource details component.

At least for now it isn't reused but I left the changes in because I think they're A Good Improvement.